src/mavlink-router/main.cpp

```diff
@@ -667,11 +672,13 @@ static int parse_confs(ConfFile &conf)
         char *addr;
         bool eavesdropping;
         unsigned long port;
+        char *filter;
     };
     static const ConfFile::OptionsTable option_table_udp[] = {
         {"address", true,   ConfFile::parse_str_dup,    OPTIONS_TABLE_STRUCT_FIELD(option_udp, addr)},
         {"mode",    true,   parse_mode,                 OPTIONS_TABLE_STRUCT_FIELD(option_udp, eavesdropping)},
         {"port",    false,  ConfFile::parse_ul,         OPTIONS_TABLE_STRUCT_FIELD(option_udp, port)},
+        {"filter",  false,  ConfFile::parse_str_dup,    OPTIONS_TABLE_STRUCT_FIELD(option_udp, filter)},
```

Comment:
I guess it would be useful to have a `ConfigFile::parse_ul_vector` and make it do the parsing. Then you just std::move<> the parsed vector instead of strdup() the string.

Reply:
this can be left for later though, as it's more intrusive.

Reply:
If you think that it can stay this way then I will not change it for now.